docs/environment/custom-domains.md

```diff
@@ -26,6 +26,8 @@ The first thing to do is register the domain in **ACM** (AWS Certificate Manager
 
 After validating the domain and the certificate we can now link the custom domain to our application via API Gateway.
 
+> Please note: This configuration is incompatible with other, more advanced settings (e.g. CloudFront) and you will need to undo this if you want to configure that instead. 
```

Comment:
I don't think we should mention that here:

- we already mention it in the CloudFront guide (in this PR)
- at this point when reading the docs you don't really want to know about possible future Cloudfront conflicts because there is nothing you can do about it right now